aten/src/THC/generic/THCTensorMathPairwise.cu

```diff
@@ -130,7 +130,7 @@ THC_API void
 THCTensor_(rshift)(THCState* state, THCTensor *self_, THCTensor *src_, real value)
 {
 #if defined(THC_REAL_IS_FLOAT) || defined(THC_REAL_IS_DOUBLE)
-  THCTensor_(mul)(state, self_, src_, pow(2, value));
+  THCTensor_(mul)(state, self_, src_, pow(2, -value));
```

Comment:
Thanks for getting to this, @vishwakftw. Could you please add a test in test_cuda.py for this?

Reply:
Sure, will do.